File: Server/src/main/content/global/skill/production/cooking/recipe/potato/impl/TunaPotato.java
package content.global.skill.production.cooking.recipe.potato.impl;

import content.global.skill.production.cooking.recipe.potato.PotatoRecipe;
import core.game.node.item.Item;

/**
 * The Tuna potato.
 */
public class TunaPotato extends PotatoRecipe {
    private static final Item TUNA_POTATO = new Item(7060);
    private static final Item TOPPING = new Item(7068);

    @Override
    public Item getProduct() {
        return TUNA_POTATO;
    }

    @Override
    public Item[] getIngredients() {
        return new Item[]{TOPPING};
    }

    @Override
    public boolean isTopping() {
        return true;
    }

    @Override
    public int getLevel() {
        return 68;
    }

    @Override
    public double getExperience() {
        return 10;
    }

}

File: Server/src/main/content/global/skill/production/cooking/recipe/potato/impl/TunaPotato.kt
package content.global.skill.production.cooking.recipe.potato.impl

import content.global.skill.production.cooking.recipe.potato.PotatoRecipe
import core.api.consts.Items
import core.game.node.item.Item

class TunaPotato : PotatoRecipe() {

    override fun getProduct(): Item {
        return TUNA_POTATO
    }

    override fun getIngredients(): Array<Item> {
        return arrayOf(TOPPING)
    }

    override fun isTopping(): Boolean {
        return true
    }

    override fun getLevel(): Int {
        return 68
    }

    override fun getExperience(): Double {
        return 10.0
    }

    companion object {
        private val TUNA_POTATO = Item(Items.TUNA_POTATO_7060)
        private val TOPPING = Item(Items.TUNA_AND_CORN_7068)
    }
}
